product_unique_serial/demo/test_demo.xml

```diff
@@ -0,0 +1,35 @@
+<?xml version="1.0" encoding="utf-8"?>
+<openerp>
+    <data>
+        
+        <!-- Products -->
+        
+        <record id="product_demo_1" model="product.product">
+            <field name="name">Huawei Ascend</field>
+            <field name="track_all">True</field>
+            <field name="categ_id" ref="product.product_category_all"/>
+            <field name="lot_unique_ok">True</field>
+            <field name="standard_price">75.50</field>
+            <field name="list_price">82.25</field>
+            <field name="qty_available">500.0</field>
```

Comment:
@sergioernestotostadosanchez 
Quitar este valor, ya que es un function, mejor afectarla por picking in